StockWatch.Backend/src/main/resources/alpaca.properties

```diff
@@ -0,0 +1,5 @@
+key_id = PKHP14LN183KRWGJMLKD
```

Comment:
use env vars

Reply:
Solved partially; Alpaca doesn't seem to accept environment variables for these. An issue will be posted on the official repository. The rest of the program does use environment variables now and profiles will be implemented at a later point.

Reply:
I mean like https://stackoverflow.com/questions/35531661/using-env-variable-in-spring-boots-application-properties

Reply:
Or see this

String keyID = "<some key ID>";
String secret = "<some secret>";
AlpacaAPI alpacaAPI = new AlpacaAPI(keyID, secret);

you can load them from spring app props?

Reply:
I haven't tried from the spring app properties just yet but from the variables themselves. That might be the issue; will try the approach of going through the main Spring properties when I continue.